src/bpmn-visu-js/mx-graph-visu.ts

```diff
@@ -0,0 +1,19 @@
+import { mxgraphFactory } from './../../node_modules/ts-mxgraph/index';
+
+const { mxClient, mxUtils } = mxgraphFactory({
+  mxLoadResources: false,
+  mxLoadStylesheets: false,
+});
+
+export const mxGraphInit = (): void => {
+  try {
+    if (!mxClient.isBrowserSupported()) {
+      mxUtils.error('Browser is not supported!', 200, false);
+    } else {
+      mxUtils.alert('Browser is supported!');
+    }
```

Comment:
Do we must to init also the mxGraphModel & the mxGraph, with empty content, here ?

Reply:
Yes, it will be the next step which I am working on now
In this PR we have just lib and simple check that it is loaded